src/sql/workbench/services/restore/common/constants.ts

```diff
@@ -10,3 +10,14 @@ export const fileFiltersSet: { label: string, filters: string[] }[] = [
 	{ label: localize('backup.filterBackupFiles', "Backup Files"), filters: ['*.bak', '*.trn', '*.log'] },
 	{ label: localize('backup.allFiles', "All Files"), filters: ['*'] }
 ];
+
+// Backup media device type: https://docs.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.management.smo.devicetype
+// Used as backupDeviceType RPC param
```

Comment:
Remove this - that's just so happens to be true but isn't necessary to call out.